README.md

```diff
@@ -0,0 +1,11 @@
+# MongoDB CSFLE Companion Projects
+
+This repository contains companion projects to go along with our [CSFLE Use Case Guide](https://docs.mongodb.com/ecosystem/use-cases/client-side-field-level-encryption-guide/). These companion projects
+are structured so that they bundle all requirements except for `mongocryptd`,
+which is available for download from the [MongoDB Download Site](https://www.mongodb.com/download-center/enterprise).
+
+For the Java implementation, see the `java` directory.
```

Comment:
Nitpick: Would be more consistent if it used "project" instead of "implementation". The "implementation" wording would make sense if you were referring to one objective rather than multiple "companion projects". Not a blocker.

Reply:
🤔